### ISSUE TITLE
[question] changelog structure

### ISSUE THREAD

**Issue by audrandessaint** [olivierlacan/keep-a-changelog#550]
I would like to know if substructures like the following one are discourage by keep a changelog ?

----
...

### Changed

Specification:

- Syntax for defining initial and final states of session (#700)

Model:

- Change representation of null messages (#643)

Generator:

- Style of Ada comments (#816)
- Detect when a generated file would overwrite an existing file (#993)
- Move operators and operations on types into separate child packages (#1126)

...

---

If so I would like to know if there is a way to handle the case when there are a lot of changes and having multiple paragraphs improves readability with the current format. If not, I would like to know why isn't it possible, and perhaps discuss the possibility to integrate it in the format.

**Comment by mudanzasIPROM** [comment]
mmmm

**Comment by ravage84** [comment]
> I would like to know if substructures like the following one are discourage by keep a changelog ?

I don't think so., though I'm not the maintainer of this changelog standard. 

If your project is big enough and typically has changes in several sub components, I would even encourage you to do so.

**Comment by joonas-fi** [comment]
If substructures are their own repos, then it could make sense for each repo to have a separate changelog

If substructures are in the same repo, then in theory one solution could be to have a separate changelog file for each component
